packages/react-navi/package.json

```
@@ -30,7 +30,7 @@
   ],
   "peerDependencies": {
     "navi": "^0.14.0",
-    "react": "^16.8.0"
+    "react": "^16.8.0 || ^17"
```

Might as well go all the way, right? 😄 
```suggestion
    "react": "^16.8.0 || ^17 || ^18"
```

---

Sure